packages/xo-web/src/common/xo/index.js

```diff
@@ -2268,15 +2268,20 @@ export const deleteSr = sr =>
 
 export const fetchSrStats = (sr, granularity) => _call('sr.stats', { id: resolveId(sr), granularity })
 
-export const forgetSr = sr =>
-  confirm({
-    title: _('srForgetModalTitle'),
-    body: _('srForgetModalMessage'),
-  }).then(() => _call('sr.forget', { id: resolveId(sr) }), noop)
+export const forgetSr = sr => forgetSrs([sr])
+
 export const forgetSrs = srs =>
   confirm({
-    title: _('srsForgetModalTitle'),
-    body: _('srsForgetModalMessage'),
+    title: _('srForgetModalTitle', { nSrs: srs.length }),
+    body: (
+      <p className='text-warning font-weight-bold'>
+        {_('srForgetModalMessage', { nSrs: srs.length })} {_('srForgetModalWarning')}
```

Comment:
The issue now is that the messages are always singular in other languages, even when you're forgetting multiple SRs. Maybe we can keep the old singular message as legacy:
```js
{srs.length < 2 ? _('srForgetModalMessage') : _('srForgetModalMessage', { nSrs: srs.length })}
```
What do you think?